### PR TITLE
New EMS Layers in Europe, plus India 

### PR DESCRIPTION
**Pull request by nickpeihl**
New layers in Kibana dashboard

To test this in local Kibana `master`, add `map.manifestServiceUrl: https://storage.googleapis.com/elastic-ems-dev-nick-catalogue/v7.0/manifest` to your `kibana.yml`.

# Europe
1. [Austria States](https://en.wikipedia.org/wiki/States_of_Austria)
1. [Belarus Regions](https://en.wikipedia.org/wiki/Regions_of_Belarus)
1. [Belgium Regions](https://en.wikipedia.org/wiki/Communities,_regions_and_language_areas_of_Belgium)
1. [Belgium Provinces](https://en.wikipedia.org/wiki/Provinces_of_Belgium)
1. [Croatia Counties](https://en.wikipedia.org/wiki/Counties_of_Croatia)
1. [Estonia Counties](https://en.wikipedia.org/wiki/Counties_of_Estonia)
1. [Hungary Counties](https://en.wikipedia.org/wiki/Counties_of_Hungary)
1. [Luxembourg Cantons](https://en.wikipedia.org/wiki/Cantons_of_Luxembourg)
1. [Poland Voivodeships](https://en.wikipedia.org/wiki/Voivodeships_of_Poland)
1. [Slovakia Regions](https://en.wikipedia.org/wiki/Regions_of_Slovakia)
1. [Slovenia Municipalities](https://en.wikipedia.org/wiki/Municipalities_of_Slovenia)

# Asia
1. [India States](https://en.wikipedia.org/wiki/Municipalities_of_Slovenia)

### CODE REVIEW

**Comment by thomasneirynck** [comment]
build was aborted due to CI instability.

retest

**Comment by thomasneirynck** [comment]
jenkins, test this

File: schema/source_schema.json [hunk 1]
```diff
@@ -18,6 +18,10 @@
   ],
   "additionalProperties": false,
   "properties": {
+    "countryCode": {
+      "type": "string",
+      "description": "Two letter country code"
+    },
```
Comment (nickpeihl) [inline schema/source_schema.json]: Country code was added to sources metadata to support future capabilities for automatic generation of visualizations such as those seen in the screenshot in this PR.

**Comment by nickpeihl** [comment]
> This is a great PR.
> 
> For posterity, a couple of things we discussed:
> 
>     * can you add some comment as to why country-code is included now?
> 
>     * we should consider having this new-layers branch (or dev-branch, or whatever we would call it), auto-deployed on GCP, just like staging and master. We need a corresponding change in the landing-page (https://github.com/elastic/ems-landing-page/blob/master/public/config.json), adding a 3rd manifest location. Then we can just use the landing-page to view this data.
> 
>     * Before merging, I'd remove the oddballs and retain the separate PRs for Brazil, USA zip codes, France. Those layer requires some manual edits and/or introduce breaking changes. It would help to keep a separate change history.

- [x] Added a comment to the PR regarding the inclusion of country codes. 
- [x] GCP auto-deployment for this feature branch is being worked on separately.
- [x] A [new issue](https://github.com/elastic/ems-landing-page/issues/79) is tracking the ems-landing-page updates for the feature branch.
- [x] Removed the countries that have their own PRs (Brazil, Portugal, Denmark, France v7, zip codes v7).

**Pull request merged**